composer.json

```diff
@@ -26,13 +26,13 @@
 		"irc": "irc://irc.freenode.net/wikidata"
 	},
 	"require": {
-		"php": ">=5.5.9",
+		"php": ">=7.2",
```

Comment:
It would be interesting to see this change connected to a ticket which justifies it. Are we expecting to use more modern language features?

In is a public library and we are dropping support for a user group without giving arguments why.
Not saying that it is very relevant in practice or that it's not the right way to go, just pointing out that we are not transparent about it.

Reply:
FTR this was not meant as blocking (there are 10 other tickets doing the same thing), it just happens to be the only one I reviewed.

Reply:
I also wonder why time is spend on changing CS tools on a library that is not actively worked on. Looks like busywork to me.

Knowing the main users of this library, I do not see the compat drop causing any issues, so why not.